### ISSUE TITLE
New submissions for Wed, 23 Aug 23

### ISSUE THREAD

**Issue by olozhika** [olozhika/ArXivDaily_StarFormation#87]
# New submissions for Wed, 23 Aug 23
Auto update Star Formation & Molecular Cloud papers at about 2:30am UTC (10:30am Beijing time) every weekday.

阅读 `Usage.md`了解如何使用此repo实现个性化的Arxiv论文推送

See `Usage.md` for instructions on how to personalize the repo. 

Keyword list: ['star formation', 'molecular cloud', 'interstellar medium', 'cloud', 'clump', 'core', 'filament', 'atomic gas', 'N-PDF']

Excluded: ['galaxies', 'galaxy cluster', 'AGN']

#### Towards an astronomical foundation model for stars with a  Transformer-based model
 - **Authors:** Henry W. Leung, Jo Bovy
 - **Subjects:** Instrumentation and Methods for Astrophysics (astro-ph.IM); Astrophysics of Galaxies (astro-ph.GA); Solar and Stellar Astrophysics (astro-ph.SR)
 - **Arxiv link:** https://arxiv.org/abs/2308.10944
 - **Pdf link:** https://arxiv.org/pdf/2308.10944
 - **Abstract**
 Rapid strides are currently being made in the field of artificial intelligence using Transformer-based models like Large Language Models (LLMs). The potential of these methods for creating a single, large, versatile model in astronomy has not yet been explored. In this work, we propose a framework for data-driven astronomy that uses the same core techniques and architecture as used by LLMs. Using a variety of observations and labels of stars as an example, we build a Transformer-based model and train it in a self-supervised manner with cross-survey data sets to perform a variety of inference tasks. In particular, we demonstrate that a $\textit{single}$ model can perform both discriminative and generative tasks even if the model was not trained or fine-tuned to do any specific task. For example, on the discriminative task of deriving stellar parameters from Gaia XP spectra, we achieve an accuracy of 47 K in $T_\mathrm{eff}$, 0.11 dex in $\log{g}$, and 0.07 dex in $[\mathrm{M/H}]$, outperforming an expert $\texttt{XGBoost}$ model in the same setting. But the same model can also generate XP spectra from stellar parameters, inpaint unobserved spectral regions, extract empirical stellar loci, and even determine the interstellar extinction curve. Our framework demonstrates that building and training a $\textit{single}$ foundation model without fine-tuning using data and parameters from multiple surveys to predict unmeasured observations and parameters is well within reach. Such "Large Astronomy Models" trained on large quantities of observational data will play a large role in the analysis of current and future large surveys.
#### The Local Value of H$_0$
 - **Authors:** Adam G. Riess, Louise Breuval
 - **Subjects:** Cosmology and Nongalactic Astrophysics (astro-ph.CO)
 - **Arxiv link:** https://arxiv.org/abs/2308.10954
 - **Pdf link:** https://arxiv.org/pdf/2308.10954
 - **Abstract**
 We review the local determination of the Hubble constant, H$_0$, focusing on recent measurements of a distance ladder constructed from geometry, Cepheid variables and Type Ia supernovae (SNe Ia). We explain in some detail the components of the ladder: (1) geometry from Milky Way parallaxes, masers in NGC 4258 and detached eclipsing binaries in the Large Magellanic Cloud; (2) measurements of Cepheids with the Hubble Space Telescope (HST) in these anchors and in the hosts of 42 SNe Ia; and (3) SNe Ia in the Hubble flow. Great attention to negating systematic uncertainties through the use of differential measurements is reviewed. A wide array of tests are discussed. The measurements provide a strong indication of a discrepancy between the local measure of H$_0$ and its value predicted by $\Lambda$CDM theory, calibrated by the cosmic microwave background ($Planck$), a decade-long challenge known as the `Hubble Tension'. We present new measurements with the James Webb Space Telescope of $>$320 Cepheids on both rungs of the distance ladder, in a SN Ia host and the geometric calibrator NGC 4258, showing reduced noise and good agreement with the same as measured with HST. This provides strong evidence that systematic errors in HST Cepheid photometry do not play a significant role in the present Hubble Tension. Future measurements are expected to refine the local determination of the Hubble constant.
#### The Large Magellanic Cloud's $\sim30$ Kiloparsec Bow Shock and its  Impact on the Circumgalactic Medium
 - **Authors:** David J. Setton, Gurtina Besla, Ekta Patel, Cameron Hummels, Yong Zheng, Evan Schneider
 - **Subjects:** Astrophysics of Galaxies (astro-ph.GA)
 - **Arxiv link:** https://arxiv.org/abs/2308.10963
 - **Pdf link:** https://arxiv.org/pdf/2308.10963
 - **Abstract**
 The interaction between the supersonic motion of the Large Magellanic Cloud (LMC) and the Circumgalactic Medium (CGM) is expected to result in a bow shock that leads the LMC's gaseous disk. In this letter, we use hydrodynamic simulations of the LMC's recent infall to predict the extent of this shock and its effect on the Milky Way's (MW) CGM. The simulations clearly predict the existence of an asymmetric shock with a present day stand-off radius of $\sim6.7$ kpc and a transverse diameter of $\sim30$ kpc. Over the past 500 Myr, $\sim8\%$ of the MW's CGM in the southern hemisphere should have interacted with the shock front. This interaction may have had the effect of smoothing over inhomogeneities and increasing mixing in the MW CGM. We find observational evidence of the existence of the bow shock in recent $H\alpha$ maps of the LMC, providing a potential explanation for the envelope of ionized gas surrounding the LMC. Furthermore, the interaction of the bow shock with the MW CGM may also explain observations of ionized gas surrounding the Magellanic Stream. Using recent orbital histories of MW satellites, we find that many satellites have likely interacted with the LMC shock. Additionally, the dwarf galaxy Ret2 is currently sitting inside the shock, which may impact the interpretation of reported gamma ray excess in Ret2. This work highlights bow shocks associated with infalling satellites are an under-explored, yet potentially very important dynamical mixing process in the circumgalactic and intracluster media.
#### Statistical Prediction of [CII] Observations by Constructing Probability  Density Functions using SOFIA, Herschel, and Spitzer Observations
 - **Authors:** Young Min Seo, Karen Willacy, Umaa Rebbapragada
 - **Subjects:** Astrophysics of Galaxies (astro-ph.GA); Instrumentation and Methods for Astrophysics (astro-ph.IM)
 - **Arxiv link:** https://arxiv.org/abs/2308.11023
 - **Pdf link:** https://arxiv.org/pdf/2308.11023
 - **Abstract**
 We present a statistical algorithm for predicting the [CII] emission from Herschel and Spitzer continuum images using probability density functions between the [CII] emission and continuum emission. The [CII] emission at 158 $\mu$m is a critical tracer in studying the life cycle of interstellar medium and galaxy evolution. Unfortunately, its frequency is in the far infrared (FIR), which is opaque through the troposphere and cannot be observed from the ground except for highly red-shifted sources (z $\gtrsim$ 2). Typically [CII] observations of closer regions have been carried out using suborbital or space observatories. Given the high cost of these facilities and limited time availability, it is important to have highly efficient observations/operations in terms of maximizing science returns. This requires accurate prediction of the strength of emission lines and, therefore, the time required for their observation. However, [CII] emission has been hard to predict due to a lack of strong correlations with other observables. Here we adopt a new approach to making accurate predictions of [CII] emission by relating this emission simultaneously to several tracers of dust emission in the same region. This is done using a statistical methodology utilizing probability density functions (PDFs) among [CII] emission and Spitzer IRAC and Herschel PACS/SPIRE images. Our test result toward a star-forming region, RCW 120, demonstrates that our methodology delivers high-quality predictions with less than 30\% uncertainties over 80\% of the entire observation area, which is more than sufficient to test observation feasibility and maximize science return. The {\it pickle} dump files storing the PDFs and trained neural network module are accessible upon request and will support future far-infrared missions, for example, GUSTO and FIR Probe.
#### The Closest View of a Fast Coronal Mass Ejection: How Faulty Assumptions  near Perihelion Lead to Unrealistic Interpretations of PSP/WISPR Observations
 - **Authors:** Ritesh Patel, Matthew J. West, Daniel B. Seaton, Phillip Hess, Tatiana Niembro, Katharine K. Reeves
 - **Subjects:** Solar and Stellar Astrophysics (astro-ph.SR)
 - **Arxiv link:** https://arxiv.org/abs/2308.11055
 - **Pdf link:** https://arxiv.org/pdf/2308.11055
 - **Abstract**
 We report on the closest view of a coronal mass ejection observed by the Parker Solar Probe (PSP)/Wide-field Imager for {Parker} Solar PRobe (WISPR) instrument on September 05, 2022, when PSP was traversing from a distance of 15.3~to~13.5~R$_\odot$ from the Sun. The CME leading edge and an arc-shaped {\emph{concave-up} structure near the core} was tracked in WISPR~field of view using the polar coordinate system, for the first time. Using the impact distance on Thomson surface, we measured average speeds of CME leading edge and concave-up structure as $\approx$2500~$\pm$~270\,km\,s$^{-1}$ and $\approx$400~$\pm$~70\,km\,s$^{-1}$ with a deceleration of $\approx$20~m~s$^{-2}$ for the later. {The use of the plane-of-sky approach yielded an unrealistic speed of more than three times of this estimate.} We also used single viewpoint STEREO/COR-2A images to fit the Graduated Cylindrical Shell (GCS) model to the CME while incorporating the source region location from EUI of Solar Orbiter and estimated a 3D speed of $\approx$2700\,km\,s$^{-1}$. We conclude that this CME exhibits the highest speed during the ascending phase of solar cycle 25. This places it in the category of extreme speed CMEs, which account for only 0.15\% of all CMEs listed in the CDAW CME catalog.

by olozhika (Xing Yuchen). 

2023-08-23
